dotnetapp-selfcontained/README.md

```diff
@@ -39,6 +39,33 @@ cd dotnetapp-selfcontained
 docker build -t dotnetapp-selfcontained -f Dockerfile.nano .
 docker run dotnetapp-selfcontained Hello .NET Core from Docker
 ```
+## Build on Windows or macOS and run the sample with Docker on Linux + ARM32 (Raspberry Pi)
+
+The goal of this section is to create and run a Docker .NET Core runtime-based image on a Raspberry Pi running Linux. The .NET Core SDK does not run on the Linux + ARM32 configuration. As a result, the instructions used for X64 don't work. There are multiple ways to get around this limitation, primarily:
+
+* Build app on X64 and copy via scp (or pscp) to ARM32 device and then build and run a Docker runtime image on the ARM32 device, or
+* Build final ARM32 image on Windows, push image to a Docker registry and then pull and run from the ARM32 device.
+
+The second option is only supported on Windows. Linux and macOS user must use the first option. For simplicity, the Windows option is provided below.
+
+The instructions assume that you are in the root of the repository.
+
+Type the following commands in Docker "Linux mode" on Windows. The instructions assume that you have a personal Docker user account called `mydockername`. You will need to change that to your actual docker account name, such as [richlander](https://hub.docker.com/u/richlander/) in the case of the author of this sample. You will also need to create a Docker repo called `dotnetapp-selfcontained-arm32`. You can create new repos in the Docker web UI.
+
+You need to be signed into the Docker client to `docker push` to Docker Hub.
+
+```console
+cd dotnetapp-prod
+docker build -t dotnetapp-selfcontained-arm32 -f Dockerfile.arm32 .
+docker tag dotnetapp-selfcontained-arm32 mydockername/dotnetapp-selfcontained-arm32
```

Comment:
Any reason to not build the image with the `mydockername/dotnetapp-selfcontained-arm32` tag in the first place?  If you want both tags you can specify multiple -t args with build.  Comment applies to dotnetapp-prod as well.